in_toto/gpg/formats.py

```diff
@@ -58,6 +58,9 @@
 import securesystemslib.formats as ssl_formats
 
 
+GPG_HASH_ALGORITHM_STRING = "pgp+SHA2"
```

Comment:
Please also fix in [docstring](https://github.com/SantiagoTorres/in-toto/blob/f3fcabb4c96f1fceaa41e93b5690e86c12746978/in_toto/gpg/formats.py#L33).

Reply:
fixed.